kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/prettypipes/shapeless.js

```diff
@@ -0,0 +1,7 @@
+onEvent('recipes', (event) => {
+    const moduletype = ['prettypipes:low_extraction_module', 'prettypipes:medium_extraction_module', 'prettypipes:high_extraction_module', 'prettypipes:low_filter_module', 'prettypipes:medium_filter_module', 'prettypipes:high_filter_module', 'prettypipes:low_retrieval_module', 'prettypipes:medium_retrieval_module', 'prettypipes:high_retrieval_module', 'prettypipes:low_crafting_module', 'prettypipes:medium_crafting_module', 'prettypipes:high_crafting_module', 'prettypipes:stack_size_module', 'prettypipes:filter_increase_modifier','ppfluids:low_fluid_extraction_module', 'ppfluids:medium_fluid_extraction_module', 'ppfluids:high_fluid_extraction_module', 'ppfluids:low_fluid_filter_module', 'ppfluids:medium_fluid_filter_module', 'ppfluids:high_fluid_filter_module', 'ppfluids:low_fluid_retrieval_module', 'ppfluids:medium_fluid_retrieval_module', 'ppfluids:high_fluid_retrieval_module'];
+
+    moduletype.forEach((module) => {
+        event.shapeless(module, [Item.of(module).ignoreNBT()]).id(module+'_clear');
```

Comment:
In the interest of consistency, we've been trying to make custom recipes very clearly 'ours' with the ID (avoids people making weird reports to the authors).

So, this should use the `id_prefix` system you see elsewhere:

```
const id_prefix = 'enigmatica:base/prettypipes/shapeless/';
```

That should go at the top of the file. 

Then your ID would look like this: 

```
.id(`${id_prefix}${module.split(':')[1]}_clearnbt`);
```
This should get you something like this as an ID

```
enigmatica:base/prettypipes/shapeless/low_extraction_module_clearnbt
```

Reply:
Cool those suggestions seem to work perfectly 
![image](https://user-images.githubusercontent.com/2611674/150416569-fdcbcfe6-a0bb-4529-8136-840090cb3068.png)

Reply:
Cool!